### PR TITLE
Move collaborators roleBinding deleteion logic from MDB to UAC

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/authservice/AuthServiceChannel.java [hunk 1]
```diff
@@ -165,6 +167,24 @@ private void initOrganizationServiceStubChannel() {
     return organizationServiceBlockingStub;
   }
 
+  private void initCollaboratorServiceStubChannel() {
```
Comment (ravishetye) [inline backend/src/main/java/ai/verta/modeldb/authservice/AuthServiceChannel.java]: this would no longer be required?

**Review by ravishetye — CHANGES_REQUESTED** [review]
clean up required, but looks good otherwise

File: backend/src/main/java/ai/verta/modeldb/authservice/AuthServiceChannel.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/authservice/AuthServiceChannel.java]: Removed

**Review by ravishetye — APPROVED** [review]

**Comment by anandjakhaniya** [comment]
Wating to approved and merge it first respective UAC PR: https://github.com/VertaAI/verta-uacservice/pull/224

**Pull request merged**